lib/widgets/progress.coffee

```diff
@@ -112,6 +118,7 @@ module.exports = class Progress
 	#
 	# @param {Object} state - progress state
 	# @param {Number} state.percentage - percentage
+	# @parm  {String} state.message - message
```

Comment:
Should also have square brackets like eta below `[state.message]`, because it's optional